nipype/interfaces/dcm2nii.py

```diff
@@ -70,13 +70,16 @@ class Dcm2nii(CommandLine):
     Examples
     ========
 
+    >>> tmp = getfixture('tmpdir')
```

Comment:
Please consider the ``.. testsetup::`` directive to hide these two lines in the documentation.

http://www.sphinx-doc.org/en/stable/ext/doctest.html#directive-testsetup

Reply:
@oesteban : do you have an example how to use it? I'm adding `.. testsetup::` before those lines and my tests are working, but everything is still visible if you ask for `help()`